### DISCUSSION TITLE
Still Supporting?

### DISCUSSION THREAD

**Discussion by HDVinnie** [CalebDW/larastan#10] [General]
Hey @calebdw 

Just curious if still maintaining this fork as noticed been opening more PRs upstream but nothing here.

**Comment by calebdw** [comment]
@HDVinnie, yep I'm still supporting! I was waiting for PHPStan to fix a bug before I released new PRs :smiley:

**Reply by HDVinnie** [reply]
Love to hear it. Thanks again. Just wanted to double check is all as switched my projects to use your fork.
